test/snippet/core/detail/all_of_type_list.cpp

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
remove newline :nail_care:

Reply:
But we always use a new line to separate seqan3 headers from other?